src/Services/Page.vala

```diff
@@ -219,18 +219,55 @@ public class ENotes.PageTable : DatabaseTable {
         }
     }
 
-    private const string[] SYMBOLS_BLACKLIST = {"#", "`", "\t", "<br>", ">", "<", "\n", "~" };
+    // Describs member of BlackList used in cleanup. reg is the regular expression and replace the string it will be replaced
+    private struct BLMember {
+        public GLib.Regex reg;
+        public string replace;
+        
+        public BLMember (GLib.Regex reg, string replace) {
+            this.reg = reg;
+            this.replace = replace;
+        }
+    
+    }
+      
     private string cleanup (string line) {
+ 
+    // This list is used for complex commands mostly by plugins e.g. "" <youtube [Link]>
+       BLMember[] Regex_complex_commands;
+       
+    // @translator this code summarises a notebook page. Instead of given youtube link this code changes into Youtube Video      
+    // [a-zA-Z0-9_\.\?\/:\=\+&\-'"]* is very greedy way for stating website but closer solutions need more space 
+       BLMember youtube_video = BLMember(/<youtube [a-zA-Z0-9_\.\?\/:\=\+&\-'"]*>/, _("Youtube Video"));
```

Comment:
This should be added to the Plugin's class :) So it can better get them.

Put a BLMember in the Plugin.vala, and equal it to null. And put a function in PluginManager to get all the regexs :)

Reply:
Whitespace before `(` https://elementary.io/docs/code/reference#whitespace.
And use `var`